dart/test/sentry_test.dart

```diff
@@ -468,6 +468,24 @@ void main() {
       expect(fixture.loggedLevel, SentryLevel.error);
     });
   });
+
+  test('calling runZonedGuarded before init does not throw', () async {
+    var expected = Exception("run zoned guarded exception");
+    Object? actual;
+
+    final completer = Completer<void>();
+    Sentry.runZonedGuarded(() {
+      throw expected;
+    }, (error, stackTrace) {
+      actual = error;
+      completer.complete();
+    });
+
+    await completer.future;
+
+    expect(actual, isNotNull);
+    expect(actual, expected);
+  });
```

Comment:
I see this test is kinda flaky, if you think it's not needed I'm also fine in removing it

Reply:
Fails on ubuntu, i'll check in a VM whats up.

Reply:
Needed to close sentry, as it was still open (active hub) because of another test with randomized test execution.